docs/SNAPTELD_CONFIGURATION.md

```diff
@@ -17,11 +17,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
```

Comment:
I'm going to defer any documentation link changes to SDI-2203. The goal for this PR is to fix the snaptel/snapteld binary, and have the installation steps match the new package location (sbin/bin).

If we have any documentation touch ups, they can be done post 0.19.0 release.